build/appveyor.yml

```diff
@@ -14,6 +14,9 @@ environment:
   # Github
   access_token:
     secure: IGs25EMrhe0au+uVDbOck6akDzC9ykvJeR0zJjhrT6idNuYOr/JpyGdefQxSlWXk
+  # Codecov
```

Comment:
You are splitting up the github section of variables.. Why move this?

Reply:
Just a mistake from when i was moving stuff around, will fix 👌